portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -102,6 +102,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
   }
 
+  private int checkUnallowedInput(String argURL, String argComment)
```

Comment:
Suggestion:

Consider returning enum value with meaning name?

For example

enum InputValidationStatus {
  VALID,
  ILLEGAL_URL,
  ILLEGAL_COMMENT
}